hay va el de regresion logistica

Rafaextremist opened the pull request:
no comentarios...

Rafaextremist committed:
espero que se pueda por veinteaba vez que lo inteto

Rafaextremist committed:
avances de regresion logistica

Rafaextremist committed:
en la funcion gradiente apenas

Rafaextremist committed:
decenso por lotes

hai va ya falta menos cada vez

Rafaextremist committed:
pequenio cambio

juliowaissman commented:
Falto la función de predicción que tenía un valor de 20 puntos. 80 puntos la tarea.

Rafaextremist committed:
termine la funcion predictor

hay le ise unos cambios por que habia error en los calculos que venian
abajo pues estos usaban theta y deveria de ser w
